packages/core/src/auth/token.ts

```diff
@@ -0,0 +1,75 @@
+import { fetchToken } from './authorize';
```

Comment:
Do we need checks in here to make sure this is only done client-side? We don't want to accidentally set `accessToken` server-side.